ui/app/AppLayouts/Communities/popups/RemotelyDestructPopup.qml

```diff
@@ -17,17 +17,26 @@ StatusDialog {
     id: root
 
     property alias model: tokenHoldersPanel.model
-
     property string collectibleName
-
+    property string accountAddress
+    property string chainName
+    property bool isFeeLoading
+    property string feeText: ""
+    property string feeErrorText: ""
+    property string feeLabel: qsTr("Remotely destruct %1 token on %2").arg(root.collectibleName).arg(root.chainName)
+    property string feePlaceholderText: qsTr("Select a hodler to see remote destruction gas fees")
+    // Account expected roles: address, name, color, emoji, walletType
+    property var accounts
```

Comment:
I think we need a signal like this here: 
`signal remotelyDestructFeesRequested(int tokenCount, var remotelyDestructTokensList, string accountAddress)` and it will be triggered when the form is ready or some changes have been done so the fee recalculation is needed again